### PR TITLE
feat: Add userscript

### PR DESCRIPTION
**Pull request by everpcpc**
For downloading from sites directly.
Only nhentai supported now.

### CODE REVIEW

File: scripts/comicbook.user.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        socket.emit('check-status', {
+            url: url,
+            start: true
+        }, function (response) {
+            checkResponse(element, url, response);
+        });
+    }
+
+
+    var galleries = document.querySelectorAll('#content .index-container .gallery');
```
Comment (shincurry) [inline scripts/comicbook.user.js]: support `My Favorites` page
support `User Profile -> Recent Favorites` page

```
    var galleries = [
        ...document.querySelectorAll('#content .index-container .gallery'),
        ...document.querySelectorAll('#content .container .gallery-favorite .gallery'),
        ...document.querySelectorAll('#content .container .gallery'),
    ];
```

File: scripts/comicbook.user.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
+// ==UserScript==
+// @name         comicbook
+// @namespace    http://comicbook.moeoverflow.com/
```
Comment (shincurry) [inline scripts/comicbook.user.js]: maybe its better to change to `https://h.comicbook.party` or `https://moeoverflow.com`

**Comment by shincurry** [comment]
that's tasty! 😋

File: scripts/comicbook.user.js (hunk 1 above)
Comment (everpcpc) [reply scripts/comicbook.user.js]: updated

**Review by shincurry — APPROVED** [review]
🥳

**Pull request merged**